gallery/src/components/NxDrawer/NxDrawerWithGlobalHeaderExample.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+
+import {
+  NxDrawer,
+  NxButton,
+  NxP,
+  NxButtonBar,
+  NxBackButton,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import { faEdit } from '@fortawesome/free-solid-svg-icons';
+
+export default function NxDrawerWithGlobalHeaderExample() {
+  const [showNarrowDrawer, setShowNarrowDrawer] = useState(false);
+  const [showNormalDrawer, setShowNormalDrawer] = useState(false);
```

Comment:
I recommend our `useToggle` utility for all of these

Reply:
> I recommend our `useToggle` utility for all of these

Fixed here: 6ebc78f1374cfaa54fc49ec651cec9bd539b4682

Reply:
> Fixed here: https://github.com/sonatype/sonatype-react-shared-components/commit/6ebc78f1374cfaa54fc49ec651cec9bd539b4682

Part of the idea there is that all of those `() => toggleDrawer()` can now just be `toggleDrawer`

Reply:
> > Fixed here: [6ebc78f](https://github.com/sonatype/sonatype-react-shared-components/commit/6ebc78f1374cfaa54fc49ec651cec9bd539b4682)
> 
> Part of the idea there is that all of those `() => toggleDrawer()` can now just be `toggleDrawer`

Fixed here: 77e4e330e6c6fb63ba0ce7eca67ec2a926a8600a